smoke-test/test_e2e.py

```diff
@@ -1239,7 +1240,7 @@ def test_generate_personal_access_token(frontend_session):
     assert "errors" in res_data  # Assert the request fails
 
 
-@pytest.mark.dependency(depends=["test_healthchecks", "test_run_ingestion"])
+# @pytest.mark.dependency(depends=["test_healthchecks", "test_run_ingestion"])
```

Comment:
don't we need that dependency?

Reply:
This is from the other PR @aditya-radhakrishnan ?